src/SALib/analyze/morris.py

```diff
@@ -364,36 +386,24 @@ def _compute_elementary_effects(
     elementary_effects = _calc_results_difference(result_increased, result_decreased)
     np.divide(elementary_effects, delta, out=elementary_effects)
     
-    def _calculate_step_size_x(input_matrix):
-        """
-        This function calculates the step of the dx at nominal values
-        """
-        max_y = np.max(input_matrix, axis=1)
-        min_y = np.min(input_matrix, axis=1)
-
-        delta = max_y - min_y
-        return delta
-    
-    see_dy = _calc_results_difference(result_increased, result_decreased)
-
-    input_matrix_nominal = _reshape_model_inputs(
-    nominal_model_inputs, num_trajectories, trajectory_size
-    )
-
-    dx_trajectory = _calculate_step_size_x(input_matrix_nominal)
-    
-    see_dy_dx = np.divide(see_dy, dx_trajectory.T)
+    if scaling==True:
+        see_dy = _calc_results_difference(result_increased, result_decreased)
```

Comment:
This is already calculated in line 386.

Reply:
Yes, but overwritten by the division on line 387, right? 
``np.divide(elementary_effects, delta, out=elementary_effects)``